src/trace-api.js

```diff
@@ -177,6 +178,22 @@ TraceAgent.prototype.runInRootSpan = function(options, fn) {
 };
 
 /**
+ * Returns a unique identifier for the current root span. This can be used to
+ * uniquely identify the current root span. If there is no current root span,
+ * or if we have lost context, this will return null. The structure and the
+ * length of the returned string should be treated opaquely - the only guarantee
+ * is that the value would unique for every root span.
+ * @returns {string} the root span id, or null if it doesn't exist
+ */
+TraceAgent.prototype.getRootSpanId = function() {
```

Comment:
We had previously discussed having `getCurrentTraceId` as the name but opted for this one because `TraceId` is an trace service concept and our agent API is built around the concept of spans. We felt this name would be more consistent with the rest of the existing API. However, for the logging use case, I think it is important that it is the trace id and not some other unique identifier so that the trace service can associate logs with the correct trace. Renaming it makes sense to me.